search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -280,6 +281,12 @@
                 (cpv/validate-date-time "revision-date end" end-date)))))
         revision-date))))
 
+(defn created-at-validation
+  [concept-type params]
+  "Validates that created-at parameter contains valid datetime strings."
+  (when-let [created-at (:created-at params)]
```

Comment:
I think we should make this validation similar to `revision-date-validation` so that we can search on multiple date ranges. Or similar to `updated-since-validation` if you only want to support after a given date. I don't have a preference as to how we want to make the `created-at` parameter behave. It's up to you.